source/landing-zone.txt

```diff
@@ -19,193 +19,165 @@ Landing Zone Design
    :depth: 2
    :class: onecol
 
-This page:
+A landing zone is a framework for establishing a well-architected and
+pre-configured cloud environment. An {+service+} landing zone defines organization-specific requirements
+for operational efficiency, security, reliability, performance, and cost,
+as well as the tools, procedures, and {+service+} configurations that teams must use to meet these requirements. 
+We recommend that all enterprise customers design a landing zone before moving workloads to MongoDB {+service+}.
+
+Designing and implementing a landing zone can help you avoid expensive 
+efforts to redesign initial setups later down the line. 
+For example, while in talks to move their on-premises databases to MongoDB {+service+}, 
+one of our recent enterprise customers voiced concerns over how peer financial services 
+companies were struggling to enforce standards for security and cost-efficiency across teams. 
+One peer had faced security risks due to inconsistent encryption policies across business units,
+while another had accumulated runaway cloud costs due to forgotten and redundant servers.
+To avoid these risks, our enterprise customer partnered with {+ps+} team
+to design a landing zone that aligns all teams and stakeholders on requirements
+including encryption at rest with {+byok+} and FinOps integrations to tag and track spending. 
 
-- Introduces the concept of a landing zone, why you need one, and the
-  considerations that influence a landing zone's design.
-- Helps you design a starter landing zone that gives
-  prescriptive guidance on:
+.. _landing-zone-considerations:
 
-  - How your teams can implement {+service+} in accordance with both
-    the {+waf+} pillars and your organization's unique requirements.
-  - How {+service+} fits into your organization's larger ecosystem and
-    architecture.
+What Are Some Landing Zone Considerations?
+------------------------------------------
 
-{+ps+} team partners with enterprise
-customers to create custom landing zones for {+service+}. If you're
-working with {+ps+}, the resources on this
-page can also help you plan for those discussions.
-
-Landing Zone Overview
----------------------
+As you create your landing zone, define requirements for the following considerations.
 
-What Is A Landing Zone?
+Critical Considerations
```

Comment:
I don't think we need to separate these into separate sections. I'm worried that users will skip over anything that's not in the critical section. We're providing all of these as recommendations. Classifying them in any way makes it that much more likely that folks will just skip the non-essential ones and diminishes the value of the recommendations in the "not critical" section.